src/App.css

```diff
@@ -126,81 +124,122 @@ body {
   padding: 0;
 }
 
-/* To-Do List Component */
-
-.input_todos{
+/* Task List Component */
 
-display: flex;
-margin: auto;
-margin-top: 20px;
-border-style: solid;
-box-shadow: 0 0 0 1px;
-width: 70%;
-height: 40px;
-font-size: 18px;
-
-}
-
-.delete_clear{
-display: flex;
-padding-left: 20px;
-padding-right: 20px;
-padding-top: 10px;
-gap: 10px;
-height: 50px;
-}
-
-button{
-font-size: 17px;
-width: 70%;
-border-radius: 10px;
-box-shadow: 0 1px;
-}
-
-.choose_weekday{
-display: flex;
-justify-content: center;
-margin-top: 50px;
+.input_todos {
+  display: flex;
+  margin: auto;
+  margin-top: 20px;
+  border-style: solid;
+  box-shadow: 0 0 0 1px;
+  width: 70%;
+  height: 40px;
+  font-size: 18px;
 }
 
-.choose_day{
-  float: left;
-  width: 40px;
+.delete_clear {
+  display: flex;
+  padding-left: 20px;
+  padding-right: 20px;
+  padding-top: 10px;
+  gap: 10px;
+  height: 50px;
 }
 
-
-.choose_month, .choose_year{
-  float: left;
-  width: 90px;
-  
+button {
+  font-size: 17px;
+  width: 70%;
+  border-radius: 10px;
+  box-shadow: 0 1px;
 }
 
-.todo_field{
+.task-list {
   display: flex;
   background-color: white;
-  margin: auto;
   justify-content: center;
   margin-top: 50px;
-  height: 440px;
+  height: 550px;
   border-style: solid;
   margin-left: 10px;
   margin-right: 10px;
   border-radius: 10px;
   box-shadow: 0 0 20px;
-  font-family: Poetsen One;
   padding: 20px;
+  background-image: url(../src/images/bulletin-board.jpg);
+}
+
+.choose_date {
+  margin-left: 97px;
+  margin-top: 15px;
+}
+
+.title_tasklist {
+  display: flex;
+  justify-content: center;
+  font-family: Poetsen One;
   text-decoration: underline;
-  background-color: #F3E779
+  margin-bottom: 20px;
+  color: black;
+}
+
+.task-list {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
 }
 
-.weekday{
+.title_tasklist {
+  display: flex;
+  justify-content: center;
+  font-family: Poetsen One;
+  text-decoration: underline;
+  margin-top: 30px;
+  font-size: 25px;
+}
+
+.value_task {
+  font-size: 20px;
+}
+
+.check {
+  display: flex;
+  justify-content: center;
   height: 20px;
-  width: 50%;
-  text-align: center;
-  font-size: 14px;
+  width: 20px;
+  margin-right: 30px;
+  margin-left: 20px;
+  margin-top: 25px;
 }
 
-.choose_date{
-  margin-left: 97px;
-  margin-top: 15px;
-}
\ No newline at end of file
+.numberoftasks {
+  display: flex;
+  margin-right: 20px;
+}
+
+.lefttasks {
+  display: flex;
+  justify-content: flex-end;
+  position: relative;
+  font-size: 20px;
+  background-color: #f3e779;
+  margin-left: 130px;
+  font-family: Poetsen One;
+}
+
+.labeltasks {
+  display: flex;
+}
+
+.task {
+  display: flex;
+  margin-top: 20px;
+  font-family: Poetsen One;
+  font-size: 22px;
+}
+
+.back {
+  background-color: #f3e779;
+  height: 100%;
+}
+
+/* Calculator Component */
```

Comment:
Why not have an extra CSS file for each component? Having all of your CSS in one file might be difficult to navigate and maintain later on.